packages/server/src/api_auth/index.ts

```diff
@@ -6,6 +6,11 @@ import { emailPasswordValidator, emailValidator } from '../validators';
 const rateLimit = require('express-rate-limit');
 
 const router = Router();
+const logoutAllSchema = z.object({
+    token: z.string().min(1), // Ensure the token property is a non-empty string
+    email: z.string().min(1), // Ensure the email property is a non-empty string
```

Comment:
We can more specific here ( email validator )

Reply:
for email that it should include @ and .com/.... types properties?